source/textInfos/__init__.py

```diff
@@ -655,7 +655,10 @@ def getMathMl(self, field):
 		@raise LookupError: If MathML can't be retrieved for this field.
 		"""
 		raise NotImplementedError
-	
+
+	def getTextInfoForCodepointMovement(self) -> Self:

```

Comment:
I think this should be a private method:

```suggestion
	def _getTextInfoForCodepointMovement(self) -> Self:
```